File: java/beecrowd 1002.java
/*
A fórmula para calcular a área de uma circunferência é: area = pi . raio^2. Considerando para este problema que pi = 3.14159:
- Efetue o cálculo da área, elevando o valor de raio ao quadrado e multiplicando por pi.
*/

import java.util.Scanner;
import java.lang.Math;

public class Main {
    public static void main(String[] args) {
        
        Scanner scan = new Scanner(System.in);
        
        double raio = scan.nextDouble();
        double areaCirculo = 3.14159 * Math.pow(raio,2);
        
        System.out.format("A=%.4f\n", areaCirculo);
        
    }
}

File: python/beecrowd 1002.py
"""
A fórmula para calcular a área de uma circunferência é: area = pi * raio^2. 
Considerando para este problema que pi = 3.14159:
  - Leia o valor do raio da circuferência;
  - Efetue o cálculo da área, elevando o valor do raio ao quadrado e multiplicando por pi.
"""

raio = float(input())

pi = 3.14159
area = pi * raio**2

print("A = {:.4f}" .format(area))
